src/Models/Character.php

```diff
@@ -46,4 +63,14 @@ public function getName(): string {
     protected function generateDisplayName() {
         $this->displayName = $this->name;
     }
+    
+    /**
+     * Sets the maximum health of a character to a given value. It also sets the
+     * health if none has been set yet.
+     * @param int $maxhealth
+     */
+    protected function setMaxhealth(int $maxhealth) {
+        $this->maxhealth = $maxhealth;
+        $this->health = $this->health??$this->maxhealth;
```

Comment:
It does only have an effect on $health if $health is NULL which is important during character creation - A character should be created at full health.